backend/main.py

```
@@ -53,66 +50,42 @@
     allow_credentials=False,
     allow_methods=["*"],
-    allow_headers=["*"]
+    allow_headers=["*"],
 )
 
+
 def get_df_by_campus(campus: int = Query(0, ge=0, le=2)):
     try:
         campus_name = campusArr[campus]
     except IndexError:
         raise HTTPException(status_code=400, detail="Invalid Campus Index")
```

Ruff (0.15.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)